### PR TITLE
Student Dashboard Changes

### LINKED ISSUE #449: Student dashboard changes
Show start time and end time for a course.
Show number of days remaining to start a course.
Should not allow a student to enroll in a course if the course is inactive.
Change timezone to Asia/Kolkata in registration page so that a student does not worry about the timezone.

### PR DESCRIPTION
**Pull request by adityacp**
This PR fixes issues mentioned in #449 
Closes #449

### CODE REVIEW

File: yaksh/templates/yaksh/quizzes_user.html [hunk 1]
```diff
@@ -39,25 +39,44 @@ <h4><b>
...
+            <div class="col-md-4">
+                {% if course.days_remain_to_start != 0 %}
+                <span class="label label-info" style="font-size: 15px">
+                    {{course.days_remain_to_start}} day(s) to start
+                </span>
+                {% endif %}
+            </div>
         </div>
+        {% if course.is_active_enrollment %}
+            <div class="alert alert-info">
```
Comment (maheshgudi) [inline yaksh/templates/yaksh/quizzes_user.html]: The UI is less than flattering.

File: yaksh/models.py [hunk 2]
```diff
@@ -722,6 +722,14 @@ def percent_completed(self, user):
             percent = round((count / len(modules)))
         return percent
 
+    def days_remain_to_start(self):
```
Comment (maheshgudi) [inline yaksh/models.py]: I don't like the method name. Could it be `days_before_start`?

File: yaksh/forms.py [hunk 3]
```diff
@@ -85,10 +85,12 @@ class UserRegisterForm(forms.Form):
     department = forms.CharField(
         max_length=64, help_text='Department you work/study at')
     position = forms.CharField(
-        max_length=64, help_text='Student/Faculty/Researcher/Industry/etc.')
+        max_length=64,
+        help_text='Student/Faculty/Researcher/Industry/Fellowship/etc.')
```
[the remark this answers is not in the document]
Comment (adityacp) [reply yaksh/forms.py]: This we can discuss and add later.

File: yaksh/forms.py [hunk 4]
```diff
@@ -85,10 +85,12 @@ class UserRegisterForm(forms.Form):
...
     timezone = forms.ChoiceField(
         choices=[(tz, tz) for tz in pytz.common_timezones],
-        initial=pytz.utc)
+        help_text='Course timings are shown based on the selected timezone',
+        initial=pytz.country_timezones['IN'][0])
```
[the remark this answers is not in the document]
Comment (adityacp) [reply yaksh/forms.py]: Same as above.

File: yaksh/templates/yaksh/quizzes_user.html (hunk 1 above)
Comment (adityacp) [reply yaksh/templates/yaksh/quizzes_user.html]: I didn't do much UI changes. Anyways we need to make better UI. I will create an issue.

**Pull request merged**